src/info/repo.rs

```diff
@@ -304,45 +307,40 @@ impl<'a> Repo<'a> {
     }
 
     pub fn get_head_refs(&self) -> Result<HeadRefs> {
-        let head = self.repo.head()?;
-        let head_oid = head.target().with_context(|| "Could not read HEAD")?;
-        let refs = self.repo.references()?;
-        let refs_info = refs
-            .filter_map(|reference| match reference {
-                Ok(reference) => match (reference.target(), reference.shorthand()) {
-                    (Some(oid), Some(shorthand)) if oid == head_oid && !reference.is_tag() => {
-                        Some(String::from(shorthand))
-                    }
-                    _ => None,
-                },
-                Err(_) => None,
+        let head_oid = self.repo.head_id().context("Could not read HEAD")?;
```

Comment:
I am getting a duplicate entry for the remote orgin:
![double_branch](https://user-images.githubusercontent.com/13710835/161147671-6c2a4290-6c8c-408a-a039-5c52f7f7c3b9.png)

Reply:
Great catch, this is fixed in 87b020e5730a3fda688190099827cbddd96a303c .